File: python/practice/largest_of_three.py
# multiple integer inputs
# num_list=list(map(int,input("Enter numbers: ").split()))
# max_num=num_list[0]
# min_num=num_list[0]
# for i in num_list:
#     if i > max_num:
#         max_num=i
#     if i < min_num:
#         min_num=i
# print(f"Largest number is {max_num}\nSmallest number is {min_num}\n")

# much better
num_list = list(map(int, input("Enter numbers: ").split()))
print("Largest:", max(num_list))
print("Smallest:", min(num_list))

File: Cpp/practice/largest_of_three.cpp
#include <iostream>
using namespace std;
int main(){
    int arr[3];
    int max,min;
    cout << "Enter Numbers: " << endl;
    for (int i=0; i < 3; i++){
        cin >> arr[i];
    }
    max=arr[0];
    min=arr[0];
    for (int i=0; i < 3; i++){
        if(arr[i]>max)
            max=arr[i];
        if(arr[i]<min)
            min=arr[i];
    }
    cout << "Max is " << max << " & Min is " << min << endl;
}